packages/mds-stream-processor/jest.config.js

```diff
@@ -1 +1 @@
-module.exports = { ...require('../../jest.config.js') }
+module.exports = { ...require('../../jest.config.js'), coverageThreshold: { global: 48 } }
```

Comment:
Note: previously there were no tests at all, so I'm not lowering coverage here. In fact, I'm raising coverage!